css/style.css

```diff
@@ -16,4 +18,293 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+p {
 
+margin-top: 1 rem;
+margin-bottom:0;
+
```

Comment:
Removing unnecessary blank lines will help keep your code clear and easy to understand.